Resources/doc/reference/introduction.rst

```diff
@@ -1,5 +1,16 @@
 Introduction
 ============
 
-This small introduction will try to explain the basics concepts behind the
-``ArticleBundle``.
+The first goal of SonataArticle Bundle is to provide a starter kit to manage contents.
+
+An ``Article`` will be like a skeleton content who will contain the main logic of what you want to build, but not the content itself. For example, this is where you can configure content types, or publication status, titles, categories...
```

Comment:
An ``Article`` is like the skeleton for a piece of content: it contains the main logic of what you want to build, but not the content itself.